src/Perlang.Tests.Integration/Operator/Binary/DivisionTests.cs

```diff
@@ -132,7 +132,7 @@ public void division_by_zero_throws_expected_runtime_error()
         [SkippableFact]
         public void division_by_zero_halts_execution()
         {
-            Skip.If(PerlangMode.ExperimentalCompilation, "Not supported in compiled mode");
+            Skip.If(PerlangMode.ExperimentalCompilation, "Division by zero has undefined behavior in compiled mode.");
```

Comment:
(Just a drive-by-improvement to make the skip message a bit clearer. We should try to do similarly on all skipped tests, ideally (or better yet, fix them so they are no longer skipped))